méli-mélo/2023-09-menu/index.html

```diff
@@ -0,0 +1,61 @@
+---
+title: Campaign Menu
+dateModified: 2023-09-11
+lang: en
+css: campaign-menu.css
+script: campaign-menu.js
+layout: no-container
+nomenu: true
+---
+        <div class="container">
+        <h1>Main content</h1>
+        <p>Example of a new menu that uses megamenu for md and lg, and GCWeb menu for sm and xs.</p>
+    </div>
+    <div style="background:#003c77;">
+        <div class="container">
+            <nav class="gcweb-menu campaign-menu" typeof="SiteNavigationElement">
+                <h2 class="wb-inv">Winterlude Site Menu</h2>
+                <button type="button" aria-haspopup="true" aria-expanded="false"><span class="wb-inv">Main </span>Menu <span class="expicon glyphicon glyphicon-chevron-down"></span></button>
+                <ul role="menu" aria-orientation="vertical">
+                    <li role="presentation"><a role="menuitem" tabindex="-1" href="home.html">Home</a></li>
+                    <li role="presentation"><a role="menuitem" tabindex="-1" href="calendar.html">Calendar</a></li>
+                    <li role="presentation"><a role="menuitem" tabindex="-1" aria-haspopup="true" aria-controls="sub-menu-1" aria-expanded="false" href="whats-happening.html">What's happening</a>
+                        <ul id="sub-menu-1" role="menu" aria-orientation="vertical">
+                            <li role="presentation"><a role="menuitem" tabindex="-1" href="5-1-test-winterlude-megaamenu-html.html">Venues</a></li>
+                        </ul>
+                    </li>
+                    <li role="presentation"><a role="menuitem" tabindex="-1" href="#">Sculptures</a></li>
+                    <li role="presentation"><a role="menuitem" tabindex="-1" aria-haspopup="true" aria-controls="sub-menu-2" aria-expanded="false" href="about.html">About</a>
+                        <ul id="sub-menu-2" role="menu" aria-orientation="vertical">
+                            <li role="presentation"><a role="menuitem" tabindex="-1" href="about.html">About Winterlude</a></li>
+                            <li role="presentation"><a role="menuitem" tabindex="-1" href="over-years.html">Over the years</a></li>
+                            <li role="presentation"><a role="menuitem" tabindex="-1" href="media.html">Media</a></li>
+                        </ul>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+    </div>
+    <div class="container mrgn-tp-xl">
+        <h1>Test content</h1>
```

Comment:
Don't use a subsequent h1 here, stay simple

```suggestion
    <div class="container">
        <h2>Fake content in this page</h2>
```